<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>300</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="grid_Layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QWidget" name="panel" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>27</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="icon">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 5px;
padding-right: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 2px;
color: white</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="hideButton">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Свернуть</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: transparent;
        border: none;
}
QPushButton:hover {
        background-color: rgba(255, 255, 255, 30);
}
QPushButton:pressed {
        background-color: rgba(255, 255, 255, 50)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="screenModeButton">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: transparent;
        border: none;
}
QPushButton:hover {
        background-color: rgba(255, 255, 255, 30);
}
QPushButton:pressed {
        background-color: rgba(255, 255, 255, 50)
}
QPushButton:checked {
        background-color: transparent;
        border: none;
}
QPushButton:checked:hover {
        background-color: rgba(255, 255, 255, 30);
}
QPushButton:checked:pressed {
        background-color: rgba(255, 255, 255, 50)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="minimumSize">
         <size>
          <width>47</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Закрыть</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
	/*icon: url(:/images/cross.svg);*/
	border: none;
}
QPushButton:hover {
	background-color: rgba(255, 0, 0, 180);
}
QPushButton:pressed {
	background-color: rgba(255, 100, 100, 220);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="leftTopField" native="true">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(cx:1, cy:1, radius:1, fx:1, fy:1, stop:0 rgba(0, 0, 0, 60), stop:0.5 rgba(0, 0, 0, 20), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QWidget" name="bottomField" native="true">
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 70), stop:0.5 rgba(0, 0, 0, 25), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="line" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="leftBottomField" native="true">
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(cx:1, cy:0, radius:1, fx:1, fy:0, stop:0 rgba(0, 0, 0, 60), stop:0.5 rgba(0, 0, 0, 20), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QWidget" name="rightField" native="true">
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 70), stop:0.5 rgba(0, 0, 0, 25), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="rightTopField" native="true">
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(cx:0, cy:1, radius:1, fx:0, fy:1, stop:0 rgba(0, 0, 0, 60), stop:0.5 rgba(0, 0, 0, 20), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="topField" native="true">
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 transparent, stop:0.5 rgba(0, 0, 0, 25), stop:1 rgba(0, 0, 0, 70));</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QWidget" name="leftField" native="true">
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 transparent, stop:0.5 rgba(0, 0, 0, 25), stop:1 rgba(0, 0, 0, 70));</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QWidget" name="rightBottomField" native="true">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(cx:0, cy:0, radius:1, fx:0, fy:0, stop:0 rgba(0, 0, 0, 60), stop:0.5 rgba(0, 0, 0, 20), stop:1 transparent);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="centralWidget" native="true">
     <layout class="QVBoxLayout" name="widgetLayout"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
